src/mapper/components/HeaderMapper.tsx

```diff
@@ -51,64 +51,64 @@ export default function HeaderMapper({
+            </div>
+            <div className="flex-1 overflow-y-auto">
+              {csvHeaders.map((header, columnIndex) => {
+                const headerMapping =
+                  currentMapping.find(
+                    (mapping) => mapping.csvColumnName === header
+                  ) ?? null;
 
-              return (
-                <HeaderMapperSelection
-                  key={columnIndex}
-                  csvHeader={header}
-                  currentMapping={headerMapping}
-                  setMapping={(headerMapping) => {
-                    const newMappings =
-                      calculateNewMappingsForCsvColumnMapingChanged(
-                        currentMapping,
-                        header,
-                        headerMapping
-                      );
+                return (
+                  <HeaderMapperSelection
+                    key={columnIndex}
+                    csvHeader={header}
+                    currentMapping={headerMapping}
+                    setMapping={(headerMapping) => {
+                      const newMappings =
+                        calculateNewMappingsForCsvColumnMapingChanged(
+                          currentMapping,
+                          header,
+                          headerMapping
+                        );
 
-                    onMappingsChanged(newMappings);
-                  }}
-                  mappingSelectionOptions={mappingSelectOptions}
-                  onMouseEnter={() => {
-                    setHoveredCsvHeader(header);
-                  }}
-                />
-              );
-            })}
+                      onMappingsChanged(newMappings);
+                    }}
+                    mappingSelectionOptions={mappingSelectOptions}
+                    onMouseEnter={() => {
+                      setHoveredCsvHeader(header);
+                    }}
+                  />
+                );
+              })}
+            </div>
+          </div>
+          <div className="flex-1 bg-gray-50">
+            <HeaderMapperDataPreview
+              examples={hoveredExamples}
+              csvHeader={hoveredCsvHeader ?? ''}
+            />
           </div>
-        </div>
-        <div className="flex-1 bg-gray-50">
-          <HeaderMapperDataPreview
-            examples={hoveredExamples}
-            csvHeader={hoveredCsvHeader ?? ''}
-          />
         </div>
       </div>
-      <div className="my-5 flex justify-between">
-        <Button variant="secondary" outline onClick={onBack}>
-          {t('mapper.back')}
-        </Button>
-        <Tooltip
-          tooltipText={t('mapper.mappingsNotValidTooltip')}
-          hidden={mapingsValid}
-        >
+      <div className="mt-auto flex-none">
+        <div className="my-5 flex justify-between">
+          <Button variant="secondary" outline onClick={onBack}>
+            {t('mapper.back')}
+          </Button>
```

Comment:
Don't we want to remove `mapper.mappingsNotValidTooltip` from translations since we removed the tooltip?